src/device/manager/mod.rs

```diff
@@ -237,6 +249,12 @@ impl DeviceManager {
                     error!("DeviceManager: Failed to return GetDeviceHandler response: {e:?}");
                 }
             }
+            Request::ModifyDevice(request) => {
+                let answer = self.modify_device(request).await;
+                if let Err(e) = actor_request.respond_to.send(answer) {
+                    error!("DeviceManager: Failed to return ModifyDevice response: {e:?}");
+                }
```

Comment:
you are using `err` in the rest of the code:

```suggestion
                if let Err(err) = actor_request.respond_to.send(answer) {
                    error!("DeviceManager: Failed to return ModifyDevice response: {err:?}");
                }
```